en/micro-integrator/docs/references/connectors/db-event-inbound-endpoint/db-event-inbound-endpoint-example.md

```diff
@@ -0,0 +1,139 @@
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<inboundEndpoint class="org.wso2.carbon.inbound.poll.dbeventlistener.DBEventPollingConsumer" name="CustomerDBEventEP" onError="eventProcessFailSeq" sequence="DBEventProcessSeq" suspend="false" xmlns="http://ws.apache.org/ns/synapse">
+    <parameters>
+        <parameter name="interval">1000</parameter>
+        <parameter name="class">org.wso2.carbon.inbound.poll.dbeventlistener.DBEventPollingConsumer</parameter>
+        <parameter name="sequential">true</parameter>
+        <parameter name="coordination">true</parameter>
+        <parameter name="inbound.behavior">polling</parameter>
+        <parameter name="driverName">com.mysql.jdbc.Driver</parameter>
+        <parameter name="url">jdbc:mysql://localhost/test</parameter>
+        <parameter name="username">root</parameter>
+        <parameter name="password">root</parameter>
+        <parameter name="tableName">CDC_CUSTOM</parameter>
+        <parameter name="filteringCriteria">byLastUpdatedTimestampColumn</parameter>
+        <parameter name="filteringColumnName">UPDATED_AT</parameter>
+        <parameter name="primaryKey">ID</parameter>
+        <parameter name="connectionValidationQuery">SELECT 1</parameter>
+        <parameter name="registryPath">dbEventIE/timestamp</parameter>
+    </parameters>
+</inboundEndpoint>
+```
+
+
+## Exporting Integration Logic as a CApp
+
+**CApp (Carbon Application)** is the deployable artefact on the Enterprise Integrator runtime. Let us see how we can export integration logic we developed into a CApp. To export the `Solution Project` as a CApp, a `Composite Application Project` needs to be created. Usually, when a solution project is created, this project is automatically created by Integration Studio. If not, you can specifically create it by navigating to  **File** -> **New** -> **Other** -> **WSO2** -> **Distribution** -> **Composite Application Project**. 
+
+1. Right click on Composite Application Project and click on **Export Composite Application Project**.</br> 
+  <img src="/assets/img/connectors/capp-project1.png" title="Export as a Carbon Application" width="300" alt="Export as a Carbon Application" />
+
+2. Select an **Export Destination** where you want to save the .car file. 
+
+3. In the next **Create a deployable CAR file** screen, select inbound endpoint and sequence artifacts and click **Finish**. The CApp will get created at the specified location provided at the previous step. 
+
+
+
+## Deploying on WSO2 Enterprise Integrator
+
+1. Navigate to [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `DB Event Listener`. Click on `DB Event Listener` and download the jar file by clicking on `Download Inbound Endpoint`. Copy this jar file into  <PRODUCT-HOME>/lib folder. 
+
+2. Download `mysql-connector-java` associated with `MySQL` server and add it to <PRODUCT-HOME>/lib folder.
```

Comment:
From where are we downloading this? Let's add that info as well just to be clear.